saleor/checkout/migrations/0041_auto_20220329_1612.py

```diff
@@ -0,0 +1,157 @@
+# Generated by Django 3.2.12 on 2022-03-29 16:12
+
+from decimal import Decimal
+
+import django.utils.timezone
+from django.db import migrations, models
+
+
+def set_default_checkout_line_currency(apps, schema_editor):
+    CheckoutLine = apps.get_model("checkout", "CheckoutLine")
+    lines = CheckoutLine.objects.select_related("checkout")
+
+    for line in lines:
+        line.currency = line.checkout.currency
+
+    CheckoutLine.objects.bulk_update(lines, ["currency"])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("checkout", "0040_add_handle_checkouts_permission"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="checkout",
+            options={
+                "ordering": ("-last_change", "pk"),
+                "permissions": (
+                    ("manage_checkouts", "Manage checkouts"),
+                    ("handle_checkouts", "Handle checkouts"),
+                    ("handle_taxes", "Handle taxes"),
```

Comment:
Adding new permission requires a separate migration that will assign given perm to all groups that already have an access to manager_checkouts. See migration of handle_checkouts as an example. https://github.com/saleor/saleor/pull/9402

Reply:
Resolved in recent commit.